app/proc/processor.go

```diff
@@ -105,6 +112,12 @@ func (p *Processor) feed(name, url, telegramChannel string, max int) {
 			continue
 		}
 
+		if (Filter{}) != filter {
```

Comment:
this check is confusing, a little bit. First of all, why this condition reversed? `filter != Filter{}` feels better. Second - such a compare with struct better be implemented as `isEmpty` member func of the filter if you want to make it ready for future additions. 

Another solution is to make it FilterTitle and just compare the string for ""